src/App.tsx

```diff
@@ -77,15 +112,57 @@ const App = (props: AppProps) => {
                         <div className="overblikk_chips">
                             <Chips>
-                                {informasjonsboksAlternativer.map((alternativ) => (
+                                {valgteInformasjonsbokser.map((alternativ) => (
                                     <Chips.Toggle
                                         key={alternativ}
-                                        selected={valgteInformasjonsbokser.includes(alternativ)}
+                                        selected={true}
                                         onClick={() => toggleComponent(alternativ)}
                                     >
                                         {alternativ}
                                     </Chips.Toggle>
                                 ))}
+
+                                {avmerketInformasjonsbokser.map((alternativ) => (
+                                    <Chips.Toggle
+                                        key={alternativ}
+                                        selected={false}
+                                        onClick={() => toggleComponent(alternativ)}
+                                    >
+                                        {alternativ}
+                                    </Chips.Toggle>
+                                ))}
+
+                                <Button
+                                    onClick={() => setInitielState(lagredeInformasjonsbokser)}
+                                    size="small"
+                                    variant="tertiary"
+                                    icon={<TrashIcon title="a11y-title" />}
+                                >
+                                    Nullstill visning
+                                </Button>
+
+                                {lagredeKnappAktiv ? (
+                                    <Button
+                                        onClick={() => {
+                                            sendOverblikkFilter(valgteInformasjonsbokser);
+                                            // Sender og fetcher parallelt? conditional fetching etter POST?
+                                            overblikkFilter.reFetch();
+                                        }}
+                                        size="small"
+                                        variant="secondary"
+                                    >
+                                        Lagre visning
+                                    </Button>
+                                ) : (
+                                    <Button
+                                        disabled={true}
+                                        size="small"
+                                        variant="secondary"
+                                        icon={<CheckmarkIcon title="a11y-title" />}
+                                    >
+                                        Lagret
+                                    </Button>
```

Comment:
I think we should try to avoid this kind of pattern where we switch out the button if an action succeeds. Mostly in order to avoid disabled-states (which [we are encouraged to avoid unless it is absolutely necessary](https://aksel.nav.no/god-praksis/artikler/deaktiverte-tilstander)).

I think an alternative where we just show a contextual message that the action (i.e. saving of "chips") succeded right next to the button is better.

Reply:
Improved